tests/profiler/bob_user_workflow_fixture.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+import pytest
+
+
+@pytest.fixture
```

Comment:
It would be nice to include a docstring describing the meaning of each persona ("alice", "bob", et al) in terms of their functionality.

Reply:
Great idea! Adding now.

Reply:
🙇